projects/swimlane/ngx-ui/src/lib/components/date-range-calendar/date-range-picker.component.html

```diff
@@ -0,0 +1,95 @@
+<ngx-dropdown [closeOnClick]="false" [closeOnOutsideClick]="true" #wrapperRef>
+  <ngx-dropdown-menu class="ngx-dropdown-menu ngx-dropdown-dark-outline" (click)="$event.stopPropagation()">
+    <div class="date-range-selection"><span>Date range selection </span></div>
+    <div class="date-range-picker">
+      <!-- Sidebar Presets -->
+      <div class="sidebar">
+        <p
+          *ngFor="let preset of presets"
+          [class.active]="selectedPreset === preset.label"
+          (click)="selectPreset(preset)"
+        >
+          {{ preset.label }}
+        </p>
+      </div>
+
+      <!-- Main Calendar + Inputs -->
+      <div class="calendar-panel">
+        <div class="input-row">
+          <div class="input-with-icon">
```

Comment:
<img width="142" alt="Screenshot 2025-07-08 at 7 37 35 AM" src="https://github.com/user-attachments/assets/264006a0-a430-4d50-bc84-43bc0fe906ee" />
 to is not vertically aligned center.